### PR TITLE
Added - BE_FE- integration with error handling

### PR DESCRIPTION
**Pull request by Anshuman1803**
Task 1: Implement frontend error handling for API calls. - Completed

Task 2: Connect authentication views to backend APIs. -Completed

### CODE REVIEW

File: Backend/config.env
Comment (lakshay927) [inline Backend/config.env:1]: email ssl port is 465 thats not backend port revert it

File: src/App.css
Comment (lakshay927) [inline src/App.css:1]: try to get rid of this css file and use tailwind

File: src/helpers/formValidation.js
Comment (lakshay927) [inline src/helpers/formValidation.js:1]: can we use it with a module for validation to make it easier

**Review by lakshay927 — CHANGES_REQUESTED** [review]
changes

File: Backend/config.env
Comment (Anshuman1803) [reply Backend/config.env:1]: DONE

File: src/App.css
Comment (Anshuman1803) [reply src/App.css:1]: DONE

File: src/helpers/formValidation.js
Comment (Anshuman1803) [reply src/helpers/formValidation.js:1]: I convert the long validation helper function into separate modules:

- isUserName : for validating the user name
- isFullName : for validating the full name
- isEmail : for validating the email address
- isPassword : for validating the password

**Review by lakshay927 — APPROVED** [review]
looks good

**Pull request merged**